Automatic Validation ended with:
> 2024-03-09T03:04:08.3559250Z ##[error] Manifest Error: Duplicate field found in the manifest. [InstallerType]

(Automated response - build 869.)

--- a/manifests/n/niconico/nair/live/latest/niconico.nair.live.installer.yaml
+++ b/manifests/n/niconico/nair/live/latest/niconico.nair.live.installer.yaml
@@ -4,7 +4,6 @@
 PackageIdentifier: niconico.nair.live
 PackageVersion: latest
 InstallerType: nullsoft
-InstallerType: exe
 InstallModes:
 - silent
 InstallerSwitches:

--- a/manifests/n/niconico/nair/live/latest/niconico.nair.live.installer.yaml
+++ b/manifests/n/niconico/nair/live/latest/niconico.nair.live.installer.yaml
@@ -6,10 +6,10 @@ PackageVersion: latest
 InstallerType: nullsoft
 InstallerType: exe
 InstallModes:
-- silentWithProgress
+- silent
 InstallerSwitches:
-  Silent: /S
-  SilentWithProgress: /s
+  Silent: /s
+  SilentWithProgress: /S
 Installers:
 - Architecture: x64
   InstallerUrl: https://n-air-app.nicovideo.jp/download/windows

--- a/manifests/n/niconico/nair/live/latest/niconico.nair.live.installer.yaml
+++ b/manifests/n/niconico/nair/live/latest/niconico.nair.live.installer.yaml
@@ -8,8 +8,7 @@ InstallerType: exe
 InstallModes:
 - silent
 InstallerSwitches:
-  Silent: /s
-  SilentWithProgress: /S
+  Silent: /S
 Installers:
 - Architecture: x64
   InstallerUrl: https://n-air-app.nicovideo.jp/download/windows

